Easy/Pascal_Triangle.java

```diff
@@ -0,0 +1,57 @@
+/*
```

Comment:
This is not JavaDoc comment. Like I mentioned in the last PR, it starts with /** and ends with */, each row should have a * at it's beginning. It should be placed right above the class name. Please refer to my previous code [here](https://github.com/FreeTymeKiyan/LeetCode-Sol-Res/blob/shuna-118/Easy/PascalsTriangle.java).